sample.mvn.parent/pom.xml

```diff
@@ -14,10 +16,43 @@
 	</parent>
 
 	<properties>
+		<!-- General Project Properties -->
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+		<!-- External Tools Properties -->
+		<bnd-bnd-maven-plugin-version>5.0.1</bnd-bnd-maven-plugin-version>
+		<m2e-lifecycle-mapping-version>1.0.0</m2e-lifecycle-mapping-version>
```

Comment:
This property can be upgraded to version 1.4.0 (the latest I found in maven repository)  and the app launches both on eclipse and using the command line, but the eclipse editor complains the version is incompatible as an error. Is this an error in eclipse? For now I will leave in version 1.0.0

Reply:
Ok maybe file a bug so that we track there that we want to do that but currently failed to do so

Reply:
In which project should I file the bug? Thanks!

Reply:
This one